### ISSUE TITLE
[Bug Report] Last patches don't work with some musics? 

### ISSUE THREAD

**Issue by Naia-love** [jagrosh/MusicBot#531]
**Describe the bug**
Seems some music doesn't work? (music that worked afaik before it was buggy again).
It just joins the channel and leave just after, and write "added _x_ (_time_) ...". If the bot already in channel playing song it'll add it to the playlist and leave when it tries to play the song that bug
note that it seems that sometime if it's not the first music (bot already playing in a channel, so bugged music added to the playlist) it do work, sometime

**To Reproduce**
tried to play this song: https://www.youtube.com/watch?v=nZLRj0OaxX0
this one don't work too:  https://youtu.be/xQBRuVTDYww 
this one too: https://www.youtube.com/watch?v=vnw8zURAxkU

**Expected behavior**
well, actually work :P

**Debug Info**
```
System Properties:
  java.version = 11.0.8
  java.vm.name = OpenJDK 64-Bit Server VM
  java.vm.specification.version = 11
  java.runtime.name = OpenJDK Runtime Environment
  java.runtime.version = 11.0.8+10-post-Ubuntu-0ubuntu120.04
  java.specification.version = 11
  os.arch = amd64
  os.name = Linux

JMusicBot Information:
  Version = 0.3.1
  Owner = 390977127153139733
  Prefix = |
  AltPrefix = null
  MaxSeconds = 0
  NPImages = false
  SongInStatus = true
  StayInChannel = false
  UseEval = true
  UpdateAlerts = true

Dependency Information:
  JDA Version = 4.2.0_212
  JDA-Utilities Version = 3.0.4
  Lavaplayer Version = 1.3.59

Runtime Information:
  Total Memory = 31
  Used Memory = 22

Discord Information:
  ID = 624910810656931842
  Guilds = 7
  Users = 3
```

**Additional context**
the console logs ( "0|music" is due to pm2)
```
0|music  | [22:26:04] [ERROR] [LocalAudioTrackExecutor]: Error in playback of nZLRj0OaxX0
0|music  | com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something broke when playing the track.
0|music  |      at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:43)
0|music  |      at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:117)
0|music  |      at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$1(DefaultAudioPlayerManager.java:349)
0|music  |      at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
0|music  |      at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
0|music  |      at java.base/java.lang.Thread.run(Thread.java:834)
0|music  | Caused by: java.lang.NullPointerException: null
0|music  |      at java.base/java.util.concurrent.ConcurrentHashMap.get(ConcurrentHashMap.java:936)
0|music  |      at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSignatureCipherManager.getCipherKeyFromScript(YoutubeSignatureCipherManager.java:136)
0|music  |      at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSignatureCipherManager.resolveFormatUrl(YoutubeSignatureCipherManager.java:103)
0|music  |      at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioTrack.loadBestFormatWithUrl(YoutubeAudioTrack.java:87)
0|music  |      at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioTrack.process(YoutubeAudioTrack.java:42)
0|music  |      at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:105)
0|music  |      ... 4 common frames omitted
```

**Comment by Coolykoen** [comment]
Been using the bot for atleast over a year now. i now have same issue as above, but even with 0.3.1.
Still the best bot i've ever used and self-hosting it makes it just so smooth :3 Any idea if / when there may be a fix? no pressure though, promise :)

**Comment by onlyyouX2** [comment]
The same error version = 0.3.1 .

**Comment by Dhvit2150** [comment]
Dont work some music not playable it is showing an error like this
14:07:59] [ERROR] [LocalAudioTrackExecutor]: Error in playback of ZYU-RdN34SM
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something broke when playing the track.
	at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:43)
	at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:117)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$1(DefaultAudioPlayerManager.java:349)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager$$Lambda$175/2114667427.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NullPointerException: null
	at java.util.concurrent.ConcurrentHashMap.get(Unknown Source)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSignatureCipherManager.getCipherKeyFromScript(YoutubeSignatureCipherManager.java:136)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSignatureCipherManager.resolveFormatUrl(YoutubeSignatureCipherManager.java:103)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioTrack.loadBestFormatWithUrl(YoutubeAudioTrack.java:87)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioTrack.process(YoutubeAudioTrack.java:42)
	at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:105)
	... 5 common frames omitted

this is showing everytime please fix these this has comed after update

**Comment by Mustachipleb** [comment]
This issue seems to be fixed in commit 2bb43b57ba271f0373f69a86271c839cf7d29463

**Comment by giovanniwijaya** [comment]
I've been using this bot since 0.2.10, and the issue was there too. But the current .jar version is virtually impossible to play music

**Comment by TeYroXOfficial** [comment]
Played a little bit with code and changing dependency of lavaplayer to 1.3.52 fixes that issue.
```
<dependency>
            <groupId>com.sedmelluq</groupId>
            <artifactId>lavaplayer</artifactId>
            <version>1.3.52</version>
        </dependency>
```
also all op yt links are working good ;)

**Comment by Naia-love** [comment]
yes i directly built master who have that changes and confirm it work :3

**Comment by giovanniwijaya** [comment]
How do I build from master? thank you

**Comment by Naia-love** [comment]
just use Maven ( `mvn clean install` ) and use the JMusic-Snapshot-All.jar ^^

**Comment by TeYroXOfficial** [comment]
You can set version in pom.xml to 0.3.1 (in line 6 to be specific) so bot won't send/show any information of new update available (until Jagrosh release 0.3.2+)

**Comment by jagrosh** [comment]
Don't do that, either wait for an official release or disable update notifications in the config. Don't change the version to match an official release.

**Comment by onlyyouX2** [comment]
> لعبت قليلاً مع الكود وتغيير تبعية lavaplayer إلى 1.3.52 يعمل على إصلاح هذه المشكلة.
> أيضا 53 يعمل بشكل جيد: P
> 
> ```
> <dependency>
>             <groupId>com.sedmelluq</groupId>
>             <artifactId>lavaplayer</artifactId>
>             <version>1.3.52</version>
>         </dependency>
> ```
> 
> أيضًا جميع روابط المرجع yt تعمل بشكل جيد ؛)

Can you send the file The rate to overcome the problem? .

**Comment by onlyyouX2** [comment]
> Played a little bit with code and changing dependency of lavaplayer to 1.3.52 fixes that issue.
> Also 53 works good :P
> 
> ```
> <dependency>
>             <groupId>com.sedmelluq</groupId>
>             <artifactId>lavaplayer</artifactId>
>             <version>1.3.52</version>
>         </dependency>
> ```
> 
> also all op yt links are working good ;)

Can you send the file The rate to overcome the problem.

**Comment by TeYroXOfficial** [comment]
> 
> 
> > Played a little bit with code and changing dependency of lavaplayer to 1.3.52 fixes that issue.
> > Also 53 works good :P
> > ```
> > <dependency>
> >             <groupId>com.sedmelluq</groupId>
> >             <artifactId>lavaplayer</artifactId>
> >             <version>1.3.52</version>
> >         </dependency>
> > ```
> > 
> > 
> > also all op yt links are working good ;)
> 
> Can you send the file The rate to overcome the problem.

Just download master and compile

**Comment by onlyyouX2** [comment]
> > > Played a little bit with code and changing dependency of lavaplayer to 1.3.52 fixes that issue.
> > > Also 53 works good :P
> > > ```
> > > <dependency>
> > >             <groupId>com.sedmelluq</groupId>
> > >             <artifactId>lavaplayer</artifactId>
> > >             <version>1.3.52</version>
> > >         </dependency>
> > > ```
> > > 
> > > 
> > > also all op yt links are working good ;)
> > 
> > 
> > Can you send the file The rate to overcome the problem.
> 
> Just download master and compile

I have modified the version, but I don't know how to rebuild

**Comment by TeYroXOfficial** [comment]
> 
> 
> > > > Played a little bit with code and changing dependency of lavaplayer to 1.3.52 fixes that issue.
> > > > Also 53 works good :P
> > > > ```
> > > > <dependency>
> > > >             <groupId>com.sedmelluq</groupId>
> > > >             <artifactId>lavaplayer</artifactId>
> > > >             <version>1.3.52</version>
> > > >         </dependency>
> > > > ```
> > > > 
> > > > 
> > > > also all op yt links are working good ;)
> > > 
> > > 
> > > Can you send the file The rate to overcome the problem.
> > 
> > 
> > Just download master and compile
> 
> I have modified the version, but I don't know how to rebuild and what do you mean by master and compile

Download master  (https://github.com/jagrosh/MusicBot) by clicking green Code, open for example in IntelliJ, on right side click Maven, then Clean and at the end Install. Output file will be in target folder and will be called JMusicBot-Snapshot-All or something like that.

**Comment by MichailiK** [comment]
Keep in mind that compiling the source code by yourself is not supported nor recommended. If you have the technical expertise, then you could try it, but we won't give any support with it.
Otherwise, please wait for an official Release.

**Comment by jagrosh** [comment]
This issue will be fixed in version 0.3.2
